src/routes/users/registerLogin.js

```diff
@@ -263,6 +263,10 @@ router.post(
         }
       });
 
+      if (Object.keys(updateObj).length === 0) {
```

Comment:
```suggestion
      if (!Object.keys(updateObj).length) {
```
Prefer truthy/falsey comparison unless we actually need to check the explicit value/type